.github/workflows/ci.yml

```diff
@@ -128,7 +142,7 @@ jobs:
         export COMPILE_RUNS=2
         export RUN_RUNS=2
 
-        ./test.sh --release --no-default-features
+        ./test.sh --release --test-rustc ${{ matrix.libgccjit_version.extra }}
```

Comment:
It seems like the `mini_test` function from the `test.sh` script is not ran anymore. Please add a function and CI job for it.

Reply:
Oh indeed, adding it.